.github/workflows/publish-collection.yml

```diff
@@ -0,0 +1,30 @@
+name: publish-collection
+on:
+  push:
+    branches:
+      - main  
+jobs:
+  build:
+    name: Publish Ansible Collection to Private Automation Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
```

Comment:
Wondering if we should add one extra step after checkout to pip install requirements.txt.

This would ensure runner env has parity with dev env and give us same ansible-galaxy version.

Reply:
Makes sense. we already do, in the lint workflow.
Added now @madorn.